client/my-sites/upgrades/checkout-thank-you/style.scss

```diff
@@ -220,3 +220,73 @@
 		max-width: 634px;
 	}
 }
+
+.purchase-detail__google-voucher-code {
+	display: flex;
+	align-items: flex-start;
+	margin-top: 12px;
+	padding-bottom: 12px;
+	border-bottom: solid 1px #E9EFF3;
+}
+
+// Google Vouche section
+.purchase-detail__google-voucher-advice {
+	margin-top: 12px;
+	display: flex;
+	max-width: 100%;
+}
+
+.dialog.card.google-voucher-dialog {
+	max-width: 530px;
+	font-size: 13px;
+	overflow: hidden;
+
+	.google-voucher-dialog__header {
```

Comment:
Since you're following our css naming practices ⭐, there's no need to nest these unless we need the added specificity to override default styles.